### ISSUE TITLE
Fails to build on Fedora 30 with ldc 1.14.0 and glib2 2.59.3 and gobject-introspection 1.59.4

### ISSUE THREAD

**Issue by Conan-Kudo** [gtkd-developers/GlibD#5]
When Fedora 30 was bumped up to ldc 1.14.0, glib2 2.59.3, and gobject-introspection 1.59.4, glibd stopped working with the following errors:

```
+ /usr/bin/meson --buildtype=plain --prefix=/usr --libdir=/usr/lib64 --libexecdir=/usr/libexec --bindir=/usr/bin --sbindir=/usr/sbin --includedir=/usr/include --datadir=/usr/share --mandir=/usr/share/man --infodir=/usr/share/info --localedir=/usr/share/locale --sysconfdir=/etc --localstatedir=/var --sharedstatedir=/var/lib --wrap-mode=nodownload --auto-features=enabled . x86_64-redhat-linux-gnu
The Meson build system
Version: 0.49.2
Source dir: /builddir/build/BUILD/GlibD-2.0.2
Build dir: /builddir/build/BUILD/GlibD-2.0.2/x86_64-redhat-linux-gnu
Build type: native build
Project name: GlibD
Project version: 2.0.2
Appending DFLAGS from environment: '-release -w -g -O1'
Appending LDFLAGS from environment: '-Wl,-z,relro'
Native D compiler: ldc2 (llvm 1.14.0 "LDC - the LLVM D compiler (1.14.0):")
Build machine cpu family: x86_64
Build machine cpu: x86_64
Found pkg-config: /usr/bin/pkg-config (1.6.0)
Dependency glib-2.0 found: YES 2.59.3
Dependency gobject-2.0 found: YES 2.59.3
Dependency gio-2.0 found: YES 2.59.3
Dependency gobject-introspection-1.0 found: YES 1.59.4
Program girtod found: YES (/usr/bin/girtod)
Message: Generating D interfaces from GIR...
Build targets in project: 3
Found ninja-1.9.0 at /usr/bin/ninja
+ /usr/bin/ninja -v -j6 -C x86_64-redhat-linux-gnu
ninja: Entering directory `x86_64-redhat-linux-gnu'
[1/332] ldc2 -I=glibd-2.0@sha -I=. -I=.. -I=generated -I=../generated -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/uuid -enable-color -release -w -g -O1 -relocation-model=pic   -of='glibd-2.0@sha/x86_64-redhat-linux-gnu_generated_gtkd_paths.d.o' -c ../x86_64-redhat-linux-gnu/generated/gtkd/paths.d
[2/332] ldc2 -I=glibd-2.0@sha -I=. -I=.. -I=generated -I=../generated -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/uuid -enable-color -release -w -g -O1 -relocation-model=pic   -of='glibd-2.0@sha/x86_64-redhat-linux-gnu_generated_glib_c_functions.d.o' -c ../x86_64-redhat-linux-gnu/generated/glib/c/functions.d
FAILED: glibd-2.0@sha/x86_64-redhat-linux-gnu_generated_glib_c_functions.d.o 
ldc2 -I=glibd-2.0@sha -I=. -I=.. -I=generated -I=../generated -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/uuid -enable-color -release -w -g -O1 -relocation-model=pic   -of='glibd-2.0@sha/x86_64-redhat-linux-gnu_generated_glib_c_functions.d.o' -c ../x86_64-redhat-linux-gnu/generated/glib/c/functions.d
../x86_64-redhat-linux-gnu/generated/glib/c/functions.d(1672): Error: glib.c.types.GUnicodeBreakType at generated/glib/c/types.d(2489) conflicts with gobject.c.types.GUnicodeBreakType at generated/gobject/c/types.d(387)
../x86_64-redhat-linux-gnu/generated/glib/c/functions.d(1679): Error: glib.c.types.GUnicodeScript at generated/glib/c/types.d(2676) conflicts with gobject.c.types.GUnicodeScript at generated/gobject/c/types.d(435)
../x86_64-redhat-linux-gnu/generated/glib/c/functions.d(1701): Error: glib.c.types.GUnicodeType at generated/glib/c/types.d(3287) conflicts with gobject.c.types.GUnicodeType at generated/gobject/c/types.d(590)
../x86_64-redhat-linux-gnu/generated/glib/c/functions.d(1706): Error: glib.c.types.GUnicodeScript at generated/glib/c/types.d(2676) conflicts with gobject.c.types.GUnicodeScript at generated/gobject/c/types.d(435)
../x86_64-redhat-linux-gnu/generated/glib/c/functions.d(1707): Error: glib.c.types.GUnicodeScript at generated/glib/c/types.d(2676) conflicts with gobject.c.types.GUnicodeScript at generated/gobject/c/types.d(435)
../x86_64-redhat-linux-gnu/generated/glib/c/functions.d(1718): Error: glib.c.types.GNormalizeMode at generated/glib/c/types.d(1192) conflicts with gobject.c.types.GNormalizeMode at generated/gobject/c/types.d(132)
[3/332] ldc2 -I=glibd-2.0@sha -I=. -I=.. -I=generated -I=../generated -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/uuid -enable-color -release -w -g -O1 -relocation-model=pic   -of='glibd-2.0@sha/x86_64-redhat-linux-gnu_generated_glib_ArrayG.d.o' -c ../x86_64-redhat-linux-gnu/generated/glib/ArrayG.d
FAILED: glibd-2.0@sha/x86_64-redhat-linux-gnu_generated_glib_ArrayG.d.o 
ldc2 -I=glibd-2.0@sha -I=. -I=.. -I=generated -I=../generated -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/uuid -enable-color -release -w -g -O1 -relocation-model=pic   -of='glibd-2.0@sha/x86_64-redhat-linux-gnu_generated_glib_ArrayG.d.o' -c ../x86_64-redhat-linux-gnu/generated/glib/ArrayG.d
generated/glib/c/functions.d(1672): Error: glib.c.types.GUnicodeBreakType at generated/glib/c/types.d(2489) conflicts with gobject.c.types.GUnicodeBreakType at generated/gobject/c/types.d(387)
generated/glib/c/functions.d(1679): Error: glib.c.types.GUnicodeScript at generated/glib/c/types.d(2676) conflicts with gobject.c.types.GUnicodeScript at generated/gobject/c/types.d(435)
generated/glib/c/functions.d(1701): Error: glib.c.types.GUnicodeType at generated/glib/c/types.d(3287) conflicts with gobject.c.types.GUnicodeType at generated/gobject/c/types.d(590)
generated/glib/c/functions.d(1706): Error: glib.c.types.GUnicodeScript at generated/glib/c/types.d(2676) conflicts with gobject.c.types.GUnicodeScript at generated/gobject/c/types.d(435)
generated/glib/c/functions.d(1707): Error: glib.c.types.GUnicodeScript at generated/glib/c/types.d(2676) conflicts with gobject.c.types.GUnicodeScript at generated/gobject/c/types.d(435)
generated/glib/c/functions.d(1718): Error: glib.c.types.GNormalizeMode at generated/glib/c/types.d(1192) conflicts with gobject.c.types.GNormalizeMode at generated/gobject/c/types.d(132)
[4/332] ldc2 -I=glibd-2.0@sha -I=. -I=.. -I=generated -I=../generated -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/uuid -enable-color -release -w -g -O1 -relocation-model=pic   -of='glibd-2.0@sha/x86_64-redhat-linux-gnu_generated_glib_AsyncQueue.d.o' -c ../x86_64-redhat-linux-gnu/generated/glib/AsyncQueue.d
FAILED: glibd-2.0@sha/x86_64-redhat-linux-gnu_generated_glib_AsyncQueue.d.o 
ldc2 -I=glibd-2.0@sha -I=. -I=.. -I=generated -I=../generated -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/uuid -enable-color -release -w -g -O1 -relocation-model=pic   -of='glibd-2.0@sha/x86_64-redhat-linux-gnu_generated_glib_AsyncQueue.d.o' -c ../x86_64-redhat-linux-gnu/generated/glib/AsyncQueue.d
generated/glib/c/functions.d(1672): Error: glib.c.types.GUnicodeBreakType at generated/glib/c/types.d(2489) conflicts with gobject.c.types.GUnicodeBreakType at generated/gobject/c/types.d(387)
generated/glib/c/functions.d(1679): Error: glib.c.types.GUnicodeScript at generated/glib/c/types.d(2676) conflicts with gobject.c.types.GUnicodeScript at generated/gobject/c/types.d(435)
generated/glib/c/functions.d(1701): Error: glib.c.types.GUnicodeType at generated/glib/c/types.d(3287) conflicts with gobject.c.types.GUnicodeType at generated/gobject/c/types.d(590)
generated/glib/c/functions.d(1706): Error: glib.c.types.GUnicodeScript at generated/glib/c/types.d(2676) conflicts with gobject.c.types.GUnicodeScript at generated/gobject/c/types.d(435)
generated/glib/c/functions.d(1707): Error: glib.c.types.GUnicodeScript at generated/glib/c/types.d(2676) conflicts with gobject.c.types.GUnicodeScript at generated/gobject/c/types.d(435)
generated/glib/c/functions.d(1718): Error: glib.c.types.GNormalizeMode at generated/glib/c/types.d(1192) conflicts with gobject.c.types.GNormalizeMode at generated/gobject/c/types.d(132)
[5/332] ldc2 -I=glibd-2.0@sha -I=. -I=.. -I=generated -I=../generated -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/uuid -enable-color -release -w -g -O1 -relocation-model=pic   -of='glibd-2.0@sha/x86_64-redhat-linux-gnu_generated_glib_c_types.d.o' -c ../x86_64-redhat-linux-gnu/generated/glib/c/types.d
[6/332] ldc2 -I=glibd-2.0@sha -I=. -I=.. -I=generated -I=../generated -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/uuid -enable-color -release -w -g -O1 -relocation-model=pic   -of='glibd-2.0@sha/x86_64-redhat-linux-gnu_generated_gtkd_Implement.d.o' -c ../x86_64-redhat-linux-gnu/generated/gtkd/Implement.d
[7/332] ldc2 -I=glibd-2.0@sha -I=. -I=.. -I=generated -I=../generated -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/uuid -enable-color -release -w -g -O1 -relocation-model=pic   -of='glibd-2.0@sha/x86_64-redhat-linux-gnu_generated_gtkd_Loader.d.o' -c ../x86_64-redhat-linux-gnu/generated/gtkd/Loader.d
ninja: build stopped: subcommand failed.
```

Any idea what's going wrong here?

**Comment by MikeWey** [comment]
Because of this change in GObject:

https://gitlab.gnome.org/GNOME/glib/commit/fc6044a4b1546c13a87985896b825f67f89861ee

**Comment by MikeWey** [comment]
The Enums are now defined in both GLib and GObject.

**Comment by MikeWey** [comment]
https://gitlab.gnome.org/GNOME/glib/merge_requests/481#note_446826

**Comment by MikeWey** [comment]
Fixed in gobject-introspection 1.59.5.
